### PR TITLE
Create Timeseries Identifier endpoint for TS rename update

### LINKED ISSUE #268: Add timeseries rename and update endpoints
The rename feature is used to change the time series identifier and the update feature is used set active flags and change the interval offset. (cwms_ts.rename_ts and cwms_ts_update_ts_id procedures respectively)

### PR DESCRIPTION
**Pull request by rma-rripken**
Fixes #268 

A TS Identifier end-point will allow rename/update and non-timewindow deletes.

### CODE REVIEW

**Comment by rma-rripken** [comment]
These changes also introduce a windowed delete for #303

File: cwms_radar_api/src/test/java/cwms/radar/api/TimeSeriesIdentifierDescriptorControllerTestIT.java [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+@Tag("integration")
+@ExtendWith(RadarApiSetupCallback.class)
+public class TimeSeriesIdentifierDescriptorControllerTestIT {
+
+    public static final String OFFICE = "SPK";
+
+    @Test
+    public void test_create_delete() throws JsonProcessingException {
+
+        int random = (int) (Math.random() * 1000);
```
Comment (MikeNeilson) [inline cwms_radar_api/src/test/java/cwms/radar/api/TimeSeriesIdentifierDescriptorControllerTestIT.java]: This needs a little explanation.

**Review by MikeNeilson — COMMENTED** [review]
Otherwise looks fine, the random int confuses me.

File: cwms_radar_api/src/test/java/cwms/radar/api/TimeSeriesIdentifierDescriptorControllerTestIT.java (hunk 1 above)
Comment (rma-rripken) [reply cwms_radar_api/src/test/java/cwms/radar/api/TimeSeriesIdentifierDescriptorControllerTestIT.java]: It should probably go away.  I had left over data from previous runs failing.  I'll remove it.

File: cwms_radar_api/src/test/java/cwms/radar/data/dao/TimeSeriesIdentifierDescriptorDaoTest.java [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+                dao.deleteAll(OFFICE, tsId);
+            }
+        }
+
+    }
+
+
+    private Location buildTestLocation() {
+
+        return new Location.Builder(locationId, "SITE", ZoneId.of("UTC"),
```
Comment (MikeNeilson) [inline cwms_radar_api/src/test/java/cwms/radar/data/dao/TimeSeriesIdentifierDescriptorDaoTest.java:95]: I should update my test improvements to handle taking a location directly.

**Review by MikeNeilson — DISMISSED** [review]
This looks good. And it gave me another use-case for the other changes I'm making for the integration tests.

**Review by MikeNeilson — APPROVED** [review]

**Pull request merged**